src/bridge/LibcoreBridge.js

```diff
@@ -135,6 +137,7 @@ const LibcoreBridge: WalletBridge<Transaction> = {
 
             const hasChanged =
               accountOps.length !== syncedOps.length || // size change, we do a full refresh for now...
+              hadSomeNullBlockHeight(accountOps) || // a bit berzerk, but in the future we will want to patch ops 1 by 1
```

Comment:
I don't think this is necessary.

```
accountOps[0].blockHeight !== syncedOps[0].blockHeight ||
```

to set after line 143 is enough

Reply:
also, blockHeight being null is a legit case (it happens on ETH, it is before a node confirm you) and we don't want to always replace again and again if the last one remains blockHeight=null.

Reply:
I don't get how it will solves the problem if ever the incorrect transaction index is `> 0`. IMO it's not worth the risk. Anyway, just made the change ¯\_(ツ)_/¯

Reply:
true. but it's a bit less likely. we just want light checks for now before we have a "better algorithm". another approach would be "for each block that was null before, check if the new one, by id, is still null, if not, need a refresh" , which is different from "one is null" which always trigger a refresh of everything at each sync